publish/publish-type.proj

```diff
@@ -72,32 +72,17 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="FindDownloadedPackagesForSigning">
+  <Target Name="PublishSymbols"
```

Comment:
We don't want to actually publish the symbols publicly at this point of the publish cycle we only want to push them to the blob feed. We will do a final publish of the full stack build where we will then publish symbols.